docs/index.md

```diff
@@ -0,0 +1,93 @@
+# Ops documentation
+
+```{toctree}
+:maxdepth: 2
+:hidden: true
+
+tutorial/index
+howto/index
+reference/index
+explanation/index
+```
+
+
+
+The Ops library (`ops`) is a Python framework for writing and testing Juju charms.
+
+> [See it on PyPI](https://pypi.org/project/ops/)
+
+The library provides:
+
+- {ref}`ops_main_entry_point`, used to initialise and run your charm
+- {ref}`ops`, the API to respond to Juju events and manage the application
+- {ref}`ops_pebble`, the Pebble client, a low-level API for Kubernetes containers
+- {ref}`ops_testing`, the recommended API for unit testing charms
+- {ref}`ops_testing_harness`, the deprecated API for unit testing charms
+
+You can structure your charm however you like, but with the `ops` library, you get a framework that promotes consistency and readability by following best practices. It also helps you organise your code better by separating different aspects of the charm, such as managing the application's state, handling integrations with other services, and making the charm easier to test.
+
+
+---------
+
+## In this documentation
+
+````{grid} 1 1 2 2
+
+```{grid-item-card} [Tutorial](tutorial/index)
+:link: tutorial/index
+:link-type: doc
+
+**Start here**: a hands-on introduction to Example Product for new users
+```
+
+```{grid-item-card} [How-to guides](/index)
+:link: howto/index
+:link-type: doc
+
+**Step-by-step guides** covering key operations and common tasks
+```
+
+````
+
+
+````{grid} 1 1 2 2
+:reverse:
+
+```{grid-item-card} [Reference](/index)
+:link: reference/index
+:link-type: doc
+
+**Technical information** - specifications, APIs, architecture
+```
+
+```{grid-item-card} [Explanation](/index)
+:link: explanation/index
+:link-type: doc
+
+**Discussion and clarification** of key topics
+```
+
+````
+
+
+---------
+
+
+## Project and community
+
+Ops is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+
+* **[Read our code of conduct](https://ubuntu.com/community/ethos/code-of-conduct)**:
+As a community we adhere to the Ubuntu code of conduct.
+
+* **[Get support](https://discourse.charmhub.io/)**:
+Discourse is the go-to forum for all questions Ops.
+
+* **[Join our online chat](https://matrix.to/#/#charmhub-ops:ubuntu.com)**:
+Meet us in the #charmhub-charmdev channel on Matrix.
+
+* **[Report bugs](https://github.com/canonical/operator/)**:
+We want to know about the problems so we can fix them.
+
+* **[Contribute docs](https://github.com/canonical/operator/tree/main/docs)**:
+The documentation sources on GitHub.
```

Comment:
@dwilding as a follow-up job to this PR, what do you think about adding a README.md file in the docs folder that explains how to contribute docs? With this change, we're basically just dropping them in a folder, which isn't super friendly.

We could move the [HACKING.md](https://github.com/canonical/operator/blob/main/HACKING.md#documentation) content (leaving a link there) and probably improve it a bunch, particularly now that everything is in git rather than Discourse (some inspiration could be taken from freesewing maybe!).

I don't think this would be part of the published docs, just something to show up when you navigate to the docs folder in GitHub or in a clone.

Reply:
Nice idea - I can work on this in a follow-up PR. Also, I'm looking at "Edit this page on GitHub" in the footer. We don't really want people to directly submit edits, right? We'd rather that people follow the process explained in HACKING.md or the new docs README.md

Reply:
If I may chime in: Imo the README and the CONTRIBUTING are repo-level thingies. As in, I wouldn't nest them under the docs. At the same time, they are a minimal manifestation of user docs and, respectively, contributor docs, which I would nest under docs for every project. Here's the incarnation of this idea for Juju: https://github.com/juju/juju/tree/3.6/docs (note the "user" and "contributor" directories and the top-level https://github.com/juju/juju/blob/3.6/docs/index.md that  introduces both.